app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -4,20 +4,15 @@ import { Alert } from "neetoui";
 
 import notesApi from "apis/notes";
 
-const DeleteAlert = ({
-  refetch,
-  onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
-}) => {
+const DeleteAlert = ({ refetch, onClose, selectedNote, setSelectedNote }) => {
   const [deleting, setDeleting] = useState(false);
-
+  const { id, title } = selectedNote;
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      await notesApi.destroy({ ids: id });
```

Comment:
The keys says `ids`, that is plural. But you are sending a string. Shouldn't you be sending an array containing a single `id`?

Reply:
Sorry @yedhink , I should have sent an array containing a single id. Should I send the as an array directly as `await notesApi.destroy({ ids: [id] })` .

Reply:
>await notesApi.destroy({ ids: [id] })

This is good enough.